pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -288,7 +288,7 @@ private Map<String, FieldIndexConfigs> calculateIndexConfigsByColName(@Nullable
     return FieldIndexConfigsUtil.createIndexConfigsByColName(tableConfig, schema, this::getDeserializer);
   }
 
-  private <C extends IndexConfig> ColumnConfigDeserializer<C> getDeserializer(IndexType<C, ?, ?> indexType) {
+  public <C extends IndexConfig> ColumnConfigDeserializer<C> getDeserializer(IndexType<C, ?, ?> indexType) {
```

Comment:
I don't think we need to make this public. See the comment in AbstractSerderIndexContract